capabilities/analytics/track-event/profile.supr

```diff
@@ -0,0 +1,39 @@
+"""
+Track Event
+Track events to better understand interactions between users and your product.
+"""
+name = "analytics/track-event"
+version = "1.0.0"
+
+"""
+Track Event
+Allows to send event to tracking system
+"""
+usecase TrackEvent unsafe {
+  input {
+    """
+    Event Name
+    A string value representing event
+    """
+    eventName! string!
+
+    """
+    Event Properties
+    Event Properties are attributes that help you define the specifics of an Event.
+    """
+    eventProperties
+  }
+
+  result string
+
+  error {
```

Comment:
Not for this PR but since this repeats so much I wonder if we can't make it somehow standard..